scripts/promote

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eEuo pipefail
+
+ROOT="$(cd "$(dirname "$0")/.." &>/dev/null; pwd -P)"
+
+if [ -z "${PROJECT_ID:-}" ]; then
+  echo "✋ Missing PROJECT_ID!" >&2
+  exit 1
+fi
+
+if [ -z "${REGION:-}" ]; then
+  echo "✋ Missing REGION!" >&2
+  exit 1
+fi
+
+if [ -z "${REVISION:-}" ]; then
+  echo "🚧 Using LATEST revision" >&2
+  REVISION="LATEST"
+fi
+
+gcloud builds submit "${ROOT}" \
```

Comment:
while we’re here - we shold also take the target percentgage, allow for gradual rollout

Reply:
Done